test/functional/inspec_exec_test.rb

```diff
@@ -52,6 +52,13 @@
     File.stat("#{outpath}/foo/bar/test.json").size.must_be :>, 0
   end
 
+  it 'can execute the exec help' do
+    out = inspec("exec --help")
```

Comment:
Could we have another test for `inspec exec help` specifically?  That's a different code path.

Reply:
I'm looking for exactly 'inspec exec help'.  I know the implementation actually treats that the same as --help, but that is not obvious to an external user. If we change the implementation, I want to know that `inspec exec help` still gives a usage message, *not* trying to find a profile named `help`.

Reply:
Added one for this as well